pkg/rhcos/ami.go

```diff
@@ -1,3 +1,5 @@
+//go:generate go run ami_regions_generate.go rhcos ../../data/data/rhcos-amd64.json ami_regions.go
```

Comment:
we should update https://github.com/openshift/installer/blob/master/hack/update-rhcos-bootimage.py to run `go generate` I will try to follow up with a PR soon